tests/plugins/test_double_plugin.py

```diff
@@ -40,6 +40,11 @@ def __len__(self):
 
 class DoublePrecisionBoringModel(BoringModel):
 
+    def __init__(self):
+        super().__init__()
+
+        self.register_buffer("complex_buffer", torch.complex(torch.rand(10), torch.rand(10)), False)
```

Comment:
You have a test for ddp + double where you use this model, but ddp does not support sync of complex valued tensors it seems (at least in 1.8). A runtime error emerges:

`E       RuntimeError: Input tensor data type is not supported for NCCL process group: ComplexFloat`

looks like you will need an independent test case for this PR.

Reply:
Thanks :smiley: I was hoping to avoid adding a test. Nevermind, fixed!